docs/content/preview/architecture/docdb-replication/async-replication.md

```diff
@@ -242,6 +242,10 @@ The following limitations apply to all xCluster modes and deployment scenarios:
 
   [Materialized views](../../../explore/ysql-language-features/advanced-features/views/#materialized-views) are not replicated by xCluster. When setting up replication for a database, materialized views need to be excluded. You can create them on the target universe after the replication is set up. When refreshing, make sure to refresh on both sides.
 
+- Truncate
+
+  The TRUNCATE command is not supported.
```

Comment:
could reference this issue for the automatic mode fix https://github.com/yugabyte/yugabyte-db/issues/23958